### PR TITLE
Update package description

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -13,38 +13,20 @@ domain-specific language (DSL).
    :alt:
 
 ⚡️ Quick Start
---------------
+---------------
```
Comment (havogt) [inline README.rst:16]: Looks wrong, but probably the emoji is 2 characters, pypi complains on upload...

File: README.rst [hunk 2]
```diff
@@ -135,9 +117,24 @@ and use an *editable* installation of GT4Py:
 Dependencies for running tests locally and for linting and formatting
 source are listed in `requirements-dev.txt`.
 
+There are notebooks in the ``examples/`` directory that can be run using
```
Comment (havogt) [inline README.rst:120]: Need to reorganize this a bit. How does the user access examples if installed from pypi?

File: README.rst [hunk 3]
```diff
@@ -13,38 +13,20 @@ domain-specific language (DSL).
...
-It is not yet released on PyPI, so users have to point to the
-git repository to install it.
 
 It is recommended to install the package in a virtual environment.
 For example:
 
 .. code-block:: bash
 
-    $ git clone https://github.com/GridTools/gt4py.git && cd gt4py
     $ python -m venv venv
```
Comment (havogt) [inline README.rst:26]: let's remove `$`

File: README.rst (hunk 2 above)
Comment (egparedes) [reply README.rst]: I added a link to the examples folder in the github repo. It's not great but at least it's something, and github renders the notebooks decently

File: README.rst (hunk 3 above)
Comment (egparedes) [reply README.rst]: Done

File: README.rst (hunk 1 above)
Comment (egparedes) [reply README.rst:14]: :+1:

**Comment by egparedes** [comment]
bors try

File: README.rst [hunk 4]
```diff
@@ -71,73 +47,73 @@ The following backends are supported:
...
 ---------------
 
-For testing GT4Py with the ``numpy`` backend, all dependencies are included in the ``setup.cfg`` and are automatically
-installed.
-The performance backends require
+For testing GT4Py with the ``numpy`` backend, all dependencies are included in the ``setup.cfg`` and are automatically installed. The performance backends also require
 
-1. `Boost <https://www.boost.org/>`__ a dependency of GridTools,
-   which needs to be installed by the user.
+1. `Boost <https://www.boost.org/>`__ library, a dependency of GridTools C++, which needs to be installed by the user.
```
Comment (havogt) [inline README.rst:52]:
```suggestion
1. the `Boost <https://www.boost.org/>`__ library, a dependency of GridTools C++, which needs to be installed by the user.
```

Comment (havogt) [reply README.rst:52]: I'll do this in the next PR.

**Comment by havogt** [comment]
bors try-

**Comment by havogt** [comment]
bors try

**Comment by havogt** [comment]
flaky test failed

**Pull request merged**